collectors/ciscoduo/collector.js

```diff
@@ -140,13 +140,13 @@ class CiscoduoCollector extends PawsCollector {
             const untilMoment = moment(parseInt(curState.maxtime));
              // Used hour-cap instead of making api call for 1 min interval, may help to reduce throtling issue.
             const { nextUntilMoment, nextSinceMoment, nextPollInterval } = calcNextCollectionInterval('hour-cap', untilMoment, this.pollInterval);
-
+            const nextPollIntervalSec = nextPollInterval > 1 ? nextPollInterval : 60;
```

Comment:
> We recommend requesting logs no more than once per minute.

Sounds like we need to have this:
```
const nextPollIntervalSec = nextPollInterval > 60 ? nextPollInterval : 60;
```

Meaning don't have a intervals less than 1 minute.